compiler/x/codegen/X86Ops.ins

```diff
@@ -1183,6 +1187,10 @@ INSTRUCTION(FLDRegReg, fld,
             BINARY(VEX_L___, VEX_vNONE, PREFIX___, REX__, ESCAPE_____, 0xd9, X87_________________(0xc0)),
             PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_SingleFP),
             PROPERTY1(IA32OpProp1_TargetRegIsImplicit)),
+INSTRUCTION(FLDDUP, fld,
+            BINARY(VEX_L___, VEX_vNONE, PREFIX___, REX__, ESCAPE_____, 0xd9, X87_________________(0xc0)),
+            PROPERTY0(IA32OpProp_ModifiesTarget | IA32OpProp_SingleFP),
```

Comment:
I don't think the `IA32OpProp_SingleFP` property makes sense unless you also define a `DLDDUP` instruction, which I don't think you need to.

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/1d77ffe76e2dc9410d223c0a72faa1f4fed3dc96..dcfb579327cf5c1dc2c7b912198553b070d81cbd)